README.md

```diff
@@ -240,6 +240,10 @@ rpc_url = "http://localhost:8545"
 strategy_type = "simple"
 [order.execution_strategy.config]
 max_gas_price_gwei = 100
+pricing_currency = "USDC"          # Display currency for quote cost breakdowns
+commission_bps = 20                 # 0.20% commission applied over subtotal
+gas_buffer_bps = 1000               # +10% buffer over estimated gas costs
+rate_buffer_bps = 14                # +0.14% buffer over base price for rate movement
```

Comment:
Should we add this to `example.toml` or `demo.toml` (`setup_anvil_nodes`) scripts as well?